### PR TITLE
TryParse API for language version no longer an extension method

### PR DESCRIPTION
**Pull request by jcouv**
### Customer scenario
If you reference the compiler nuget package, an extension on the `string` type becomes available. That has been reported as being annoying (it adds an unexpected completion).
I'm making this API a regular (non-extension) method to remedy this. 

### Bugs this fixes
Fixes https://github.com/dotnet/roslyn/issues/26132

### Performance impact
Low. Just make the the API non-extension.

### CODE REVIEW

**Comment by etbyrd** [comment]
@dotnet-bot test windows_debug_vs-integration_prtest

**Comment by jcouv** [comment]
@dotnet/roslyn-compiler One-liner for review. Thanks

**Comment by jcouv** [comment]
From discussion with @cston, I'll ping compat council to see if we can just take the break (not make it an extension method).

File: src/Compilers/CSharp/Portable/LanguageVersion.cs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.ComponentModel;
```
Comment (cston) [inline src/Compilers/CSharp/Portable/LanguageVersion.cs]: `using` is no longer needed.

**Comment by jcouv** [comment]
@dotnet/roslyn-ide @jinujoseph for a tiny review. I had to update a few lines of IDE code/tests to adjust to this change to a compiler API. Thanks

**Comment by jcouv** [comment]
@dotnet/roslyn-ide @jinujoseph for a tiny review. I had to update a few lines of IDE code/tests to adjust to this change to a compiler API. Thanks

**Comment by jasonmalinowski** [comment]
@jcouv PR descripition no longer reflects the content? I see comments about EditorBrowsable.Never but don't see that in the code.

I presume this has gotten whatever breaking change signoffs you need?

**Comment by jcouv** [comment]
@jasonmalinowski Thanks. Updated the description.
Yes, the compat council approved this breaking change.

@jinujoseph for ask-mode approval for 15.8. Thanks

**Comment by jinujoseph** [comment]
Approved to merge from IDE side for 15.8.Preview4

**Comment by jcouv** [comment]
Thanks
I'm approving from compiler side. FYI @jaredpar

**Pull request merged**